tests/queries/0_stateless/03267_arrow_flight_descriptor_type_setting.sql

```diff
@@ -0,0 +1,15 @@
+-- Test that the arrow_flight_request_descriptor_type setting exists and accepts valid values
+
+-- Test default value
+SELECT name, value, changed, description FROM system.settings WHERE name = 'arrow_flight_request_descriptor_type';
+
+-- Test setting to 'path'
+SET arrow_flight_request_descriptor_type = 'path';
+SELECT value FROM system.settings WHERE name = 'arrow_flight_request_descriptor_type';
+
+-- Test setting to 'command'
+SET arrow_flight_request_descriptor_type = 'command';
+SELECT value FROM system.settings WHERE name = 'arrow_flight_request_descriptor_type';
+
+-- Test that invalid value throws error (should fail)
+SET arrow_flight_request_descriptor_type = 'invalid'; -- { serverError BAD_ARGUMENTS }
```

Comment:
this test does not test the issue this PR fixes - we need an integration test for this to make sure setting works properly with Dremio... though I'm not sure we have Dremio installed - probably we need to deploy it into our test environment